### PR TITLE
add loading failed for 'i386 error solution

### PR DESCRIPTION
**Pull request by DejanDraschkow**

### CODE REVIEW

**Comment by dengemann** [comment]
Can you carry over these changes to the README.Rmd ? The README.md is automatically generated form the Rmd.

File: README.md [hunk 1]
```diff
@@ -152,6 +152,18 @@ plt$show()  # show the figure, then search for the window popping up.
...
+    `loading failed for 'i386'` error.
+    To force building the package for your currently running R version
+    use the `INSTALL_opts` argument of `install_github`:
+
+<!-- end list -->
+
+``` r
+devtools::install_github("mne-tools/mne-r", INSTALL_opts=c("--no-multiarch"))
+```
+
```
Comment (dengemann) [inline README.md:166]: Please put this in README.Rmd.

File: README.md [hunk 2]
```diff
@@ -152,6 +152,18 @@ plt$show()  # show the figure, then search for the window popping up.
   - Some of the examples may depend on the latest MNE-Python code. We
     will try to provide patches little by little in MNE-Python or here.
 
+  - Currently, if both 32-bit and 64-bit versions of R are installed, 
+    it seems devtools tries to build for both of them. This yields an
```
Comment (dengemann) [inline README.md:156]: an -> a

**Comment by DejanDraschkow** [comment]
ah, yes! just saw that - sorry!

File: README.md (hunk 2 above)
Comment (dengemann) [inline README.md:156]: remove: both

**Review by dengemann — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DejanDraschkow) [reply README.md:166]: done in the new PR

File: README.md (hunk 2 above)
Comment (DejanDraschkow) [reply README.md:156]: done in the new PR

**Pull request closed without merging**